src/libraries/System.Runtime/tests/LinkerTests/System.Runtime.LinkerTests.proj

```diff
@@ -0,0 +1,5 @@
+<Project DefaultTargets="Build">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove(Directory.Build.props))" />
+
+  <Import Project="$([MSBuild]::GetPathOfFileAbove(Directory.Build.targets))" />
+</Project>
```

Comment:
I think this project should not just be the entry point to drive the generation, restore and execution but also define the source files. Consider using [Microsoft.Build.NoTargets](https://github.com/dotnet/runtime/blob/master/src/libraries/shims/ApiCompat.proj#L1)  instead and use `EnableDefaultItems` to access the `Compile` items.

Reply:
That is interesting point I didn't think of that. I will leave this as is for now so that we can merge and unblock other folks from adding tests which are waiting for this, but I will explore that and change this model.